test/unit/firebase-app.spec.ts

```diff
@@ -496,7 +497,7 @@ describe('FirebaseApp', () => {
       expect(gcsNamespace).not.be.null;
     });
 
-    it('should return a cached version of Messaging on subsequent calls', () => {
+    it('should return a cached version of Storage on subsequent calls', () => {
```

Comment:
Fixed a typo. I can move this to a separate PR if we prefer that :)

Reply:
Thanks 👍